File: Java/1901-1950/1929. Concatenation of Array.java
// https://leetcode.com/problems/concatenation-of-array/

class Solution {
    public int[] getConcatenation(int[] nums) {
        int size = nums.length;
        int[] result = new int[size*2];

        for(int index=0; index<size; index++){
            result[index] = nums[index];
            result[index+size] = nums[index];
        }

        return result;
    }
}

File: Python/1901-1950/1929. Concatenation of Array.py
# https://leetcode.com/problems/concatenation-of-array/

class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        # Easy Solution
        # return nums + nums
        
        size = len(nums)
        result = []

        for index in range(0, 2*size):
            result.append(nums[index%size])

        return result
